helm/microcosm/templates/cgimap-deployment.yaml

```diff
@@ -22,18 +22,21 @@ spec:
         release: {{ .Release.Name }}
         run: {{ .Release.Name }}-cgimap
     spec:
-
       containers:
         - name: {{ .Chart.Name }}-cgimap-backend
           {{- if eq .Values.cloudProvider.name  "minikube" }} 
           image: {{ .Values.ingestion.pods.cgimap.image.repository }}:{{ .Values.ingestion.pods.cgimap.image.tag }}
           {{- else }}
           image: {{ .Values.cloudProvider.dockerRegisteryUrl }}/{{ .Values.ingestion.pods.cgimap.image.repository }}:{{ .Values.ingestion.pods.cgimap.image.tag }}
           {{- end }}
-          # ports:
-          #   - name: fcgi
-          #     containerPort: 8000
-          #     protocol: TCP
+          command:
+            - "/app/start.sh"
+          args:
+            - "--port=8000"
+            - "--instance=30"
```

Comment:
Make it so this value can also be changed.

Reply:
Done